<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Leds</class>
 <widget class="QDialog" name="Leds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="leds_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Led 1 On/Off</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="leds_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Led 2 On/Off</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="leds_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Led 3 On/Off</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="leds_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Led 4 On/Off</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="all_led">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>All Led On/Off</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="led_start_stop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>168</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Led Start/Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
